src/Avalonia.Themes.Fluent/ToolTip.xaml

```diff
@@ -1,17 +1,74 @@
-    <ControlTemplate>
-      <ContentPresenter Name="PART_ContentPresenter"
-                        Background="{TemplateBinding Background}"
-                        BorderBrush="{TemplateBinding BorderBrush}"
-                        BorderThickness="{TemplateBinding BorderThickness}"
-                        ContentTemplate="{TemplateBinding ContentTemplate}"
-                        Content="{TemplateBinding Content}" 
-                        Padding="{TemplateBinding Padding}"/>
-    </ControlTemplate>
-  </Setter>
-</Style>
\ No newline at end of file
+<Styles xmlns="https://github.com/avaloniaui" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+  <Design.PreviewWith>
+      <Grid RowDefinitions="Auto,Auto"
+            ColumnDefinitions="Auto,Auto"
+            HorizontalAlignment="Center">
+        <Border Grid.Column="0"
+                Grid.Row="1"
+                Background="{DynamicResource ThemeAccentBrush}"
+                Margin="5"
+                Padding="50"
+                ToolTip.Tip="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.">
+          <TextBlock>Hover Here</TextBlock>
+        </Border>
+        <CheckBox Grid.Column="1"
+                  Margin="5"
+                  Grid.Row="0"
+                  IsChecked="{Binding ElementName=Border, Path=(ToolTip.IsOpen)}"
+                  Content="ToolTip Open" />
+        <Border Name="Border"
+                Grid.Column="1"
+                Grid.Row="1"
+                Background="{DynamicResource ThemeAccentBrush}"
+                Margin="5"
+                Padding="50"
+                ToolTip.Placement="Bottom">
+          <ToolTip.Tip>
+            <StackPanel>
+              <TextBlock Classes="h1">ToolTip</TextBlock>
+              <TextBlock Classes="h2">A control which pops up a hint when a control is hovered</TextBlock>
+            </StackPanel>
+          </ToolTip.Tip>
+          <TextBlock>ToolTip bottom placement</TextBlock>
+        </Border>
+      </Grid>
+  </Design.PreviewWith>
+
+  <Style Selector="ToolTip">
+    <Setter Property="Foreground" Value="{DynamicResource ToolTipForeground}" />
+    <Setter Property="Background" Value="{DynamicResource ToolTipBackground}" />
+    <Setter Property="BorderBrush" Value="{DynamicResource ToolTipBorderBrush}" />
+    <Setter Property="BorderThickness" Value="{DynamicResource ToolTipBorderThemeThickness}" />
+    <Setter Property="FontFamily" Value="{DynamicResource ContentControlThemeFontFamily}" />
+    <Setter Property="FontSize" Value="{DynamicResource ToolTipContentThemeFontSize}" />
+    <Setter Property="Padding" Value="{DynamicResource ToolTipBorderThemePadding}" />
+    <Setter Property="Transitions">
+      <Transitions>
+        <DoubleTransition Property="Opacity" Duration="0:0:0.2"/>
```

Comment:
It works when ToolTip is opening, but doesn't, when it is hiding.
Not sure, how it should be done with Avalonia animations.

Also "0:0:0.2" is approximated value, because I couldn't found FadeInThemeAnimation duration from UWP. I need to try read it in runtime later from UWP app.